Source/RapyutaSimulationPlugins/Private/Robots/RRRobotROS2Interface.cpp

```diff
@@ -297,8 +297,8 @@ void URRRobotROS2Interface::MovementCallback(const UROS2GenericMsg* Msg)
         // probably should not stay in msg though
         FROSTwist twist;
         twistMsg->GetMsg(twist);

```

Comment:
what is the difference between these two initialization?

Reply:
No difference actually, both calling the copy constructor, it's just easier to read with `=`
https://www.geeksforgeeks.org/copy-constructor-vs-assignment-operator-in-c/#:~:text=copy%20constructor%2C-,same,-as%20Test%20t3